build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
It seems like that `docker exec` doesn't run with root-rigths by default anyway. There`s a separate `--privileged` flag to get more rights. I'm searching for more information.